src/usb.rs

```diff
@@ -15,22 +15,40 @@ use crate::gpio::gpioa::{PA11, PA12};
 use crate::gpio::{PushPull, AF14};
 pub use stm32_usbd::UsbBus;
 
+/// Trait implemented by all pins that can be the "D-" pin for the USB peripheral
+pub trait DMPin: crate::private::Sealed {}
```

Comment:
```suggestion
pub trait DmPin: crate::private::Sealed {}
```

Because of https://github.com/stm32-rs/stm32f3xx-hal/blob/2da75a7dff397057ffe0bd82abc690c9a88af3b9/src/lib.rs#L89 this is allowed, but let's try to stick to CamelCase where possible. This also applies to `DM` -> `Dm` later in the file.

Reply:
Thanks!